source/nodejs/adaptivecards-designer-app/src/app.ts

```diff
@@ -17,6 +17,7 @@ window.onload = function () {
     ACDesigner.GlobalSettings.enableDataBindingSupport = true;
     ACDesigner.GlobalSettings.showDataStructureToolbox = false;
     ACDesigner.GlobalSettings.showSampleDataEditorToolbox = true;
```

Comment:
Possible need to make the new editor open. It's in `+` collapse state in my machine

![image](https://user-images.githubusercontent.com/6290692/180305063-290048f1-cb14-4b7d-9b4c-61eff11733e8.png)

Reply:
I updated it so that we properly restore the state when we include the host data editor :)